scss/partials/_base.scss

```diff
@@ -160,14 +160,30 @@ time {
 ul {
   margin: 0;
   font-weight: inherit;
+  padding-left: 1.5rem;
+
+  li {
+    list-style: none;
+
```

Comment:
Nice work here. May I suggest to add this rule `@extend p,.p--body;`  This will account for instances of li without p tags, If that makes sense.

Reply:
So theres's this weird thing about `li` tag and the font we use. When I basically add the font to `li` tag, it change the `li::before` bullet's shape from circle `content: "\2022"` to square bullet. My workaround is to extend the p style to just `span` tag in `li` tag

Reply:
Will the`li` look correct without any `span` or `p` nested within?

I'm just thinking about how content editors will enter text in a blog post or any other copy on the VT site.

Another workaround is to set a rule in `::before` 
  `@extend p,.p--body;`
   ` content: "\2022";`
  `  font-family: none;`
  
  follow by: `
  li::marker {
    content: '';
  }`
  
 But if your code accomplishes the fix then I'm happy.

Reply:
I'm loving this workaround. However, for the `li::marker` I prefer to use `list-style: none` just to not seeing it on the html tree at all for a neater workspace. I'll only extend `p` tag here since `.p--body` is pretty much the same and I would have to move the partial `_typography.scss` in `style.scss` to the top for it to take effect.

Reply:
Sweet, thank you.